packages/account/src/Components/forms/form-select-field.tsx

```diff
@@ -0,0 +1,82 @@
+import { Autocomplete, SelectNative } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Field, FieldProps, FormikErrors } from 'formik';
+import React from 'react';
+import { TGetField, TListItem } from '../additional-kyc-info-modal/form-config';
+
+type TProps = TGetField & {
+    onItemSelection?: (item: TListItem) => void;
+    list_height?: string;
+};
+
+const FormSelectField: React.FC<TProps> = ({
+    label,
+    name,
+    required = false,
+    disabled = false,
+    list_items,
+    onItemSelection,
+    placeholder,
+    list_height,
+}) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const onSelect =
+        (
+            field: string,
+            setFieldValue: (
+                field: string,
+                value: string,
+                shouldValidate?: boolean
+            ) => Promise<void | FormikErrors<Record<string, string>>>
+        ) =>
+        ({ value, text }: TListItem) => {
+            setFieldValue(field, value ? text : '', true);
+        };
+    return (
+        <Field name={name}>
+            {({ field, meta: { touched, error }, form: { setFieldValue } }: FieldProps<string>) => (
+                <React.Fragment>
+                    {!is_mobile && (
+                        <Autocomplete
+                            {...field}
+                            disabled={disabled}
+                            label={label}
+                            // @ts-expect-error This needs to fixed in AutoComplete component
+                            list_items={list_items}
+                            placeholder={placeholder}
+                            required={required}
+                            data-lpignore='true'
+                            autoComplete='off' // prevent chrome autocomplete
+                            // @ts-expect-error This needs to fixed in AutoComplete component
```

Comment:
[Question]
- Is there a tech debt card to type the shared components that you've mentioned here?
- Do you need to add a TODO to remove these later on?

Reply:
There's no tech debt card for this yet there'll be a TS error if the component expects the same props being passed here: TS will throw an error by itself

Reply:
Help to add a TODO to remove these once someone address the type issues in the technical debt that you're going to create here https://app.clickup.com/20696747/v/l/li/900502075346